legal-api/tests/unit/reports/test_report.py

```diff
@@ -183,3 +187,102 @@ def test_get_pdf(session, test_name, identifier, entity_type, report_type, templ
     assert filename
     template = report._get_template()
     assert template
+
+
+def test_alteration_name_change_to_numbered_company(session):
+    """Assert alteration name change filings can be returned as a PDF."""
+    numbered_company_name = '1234567 B.C. Ltd.'
+    named_company_name = 'New Name Ltd.'
+    identifier = 'BC1234567'
+    entity_type = 'BEN'
+    report_type = 'certificateOfNameChange'
+
+    # An existing business
+    business = factory_business(identifier=identifier, entity_type=entity_type)
+
+    # changes its name to a named company
```

Comment:
Did you mean to remove this code block? It looks like the procedure below (lines 234+) are set up to do this part of the test, but actually THIS procedure seems to do both parts already :)

Reply:
I'm not sure if I understood it but what I'm trying to do is to simulate creating a business, then filing and update the business name twice: to a named company and to a numbered company.

Reply:
It looks like you have a unit test procedure below this one that does only the "named" test. So I thought THIS procedure would do only the "numbered" test (but it does both).

Reply:
Got it. I just wanted to split the tests into 2 different test cases but yes, we could do it in only one that changes its name to a named company and then back to a numbered one.

Reply:
I like 2 separate tests/cases.

Reply:
Is it a valid scenario: to change a name from a numbered company to a different numbered company?

Reply:
I don't believe that's a valid scenario but you can confirm with a BA (Sienna or Linda).

My initial comment with this code is that the second unit test looks like the first unit test with 1 line removed. My guess was that you meant to have 2 unit tests (one for each scenario) but you forgot to remove the "name" code from the first test. But I am totally confused, please identify what each unit test procedure verifies and why they look the same but aren't the same.

Reply:
Yes, I forgot to remove the "named company name". 

They look similar because I supposed a name change to a numbered company would only happen after passing through changing the name to a named company before. 

Numbered company test:
- Changes its name to a named company
- Changes its name to a numbered company
- Does asserts

Named company test:
- Changes its name to a named company
- Does asserts

I can change to only one test that does the asserts every time it changes the company name.

I'm waiting for Sienna to confirm.

Reply:
Oh! In the first test, "change its name to a named company" is setup for the second change? Please add a code comment to make this clear. Thanks :)